cli/utilities/utils.py

```diff
@@ -1,6 +1,8 @@
 import os
 import re
+from threading import Thread
 
+from ceph.ceph import CommandFailed
```

Comment:
better to add new `CommandFailed` exception here

Reply:
The logic I used here Vaibhav was to catch the CommandFailed exception thrown by the exec_command. So we need to use the same right? Adding a new exception specific to this file will not help in handling the above requirement. Please correct me if I missed something.

Reply:
+1